src/translations/sqlb_de.ts

```diff
@@ -2881,7 +2881,7 @@ Sie können SQL-Statements aus einer Objektzeile fassen und in anderen Anwendung
     <message>
         <location filename="../MainWindow.ui" line="781"/>
         <source>&amp;Recent Files</source>
-        <translation>&amp;Kürzliche Dateien</translation>
+        <translation>&amp;Zuletzt geöffnete Dateien</translation>
```

Comment:
This replaces the keyboard shortcut. Have you verified that there ideally are no duplicates?